src/components/SearchBar/searchbar.js

```diff
@@ -0,0 +1,312 @@
+
+    search.addEventListener('click', async (e) => {
+      e.preventDefault()
+      const send = {
+        lang: 'fr',
+        strict: false,
+        zipcode: '',
+        cities: tags_set.city,
+        typographies: tags_set.type,
+        styles: tags_set.style,
+        intervenants: tags_set.architect,
+        streets: tags_set.street
+      }
+
+      // const send = {
+      //   lang: 'fr',
+      //   strict: false,
+      //   zipcode: null,
+      //   city: null,
+      //   typology: null,
+      //   style: null,
+      //   architects: null,
+      //   streets: null
+      // }
+
+      // TOT DELETE WHEN MULTIPLE VALUES ARE POSSIBLE
+      if (tags_set['zip code'].length !== 0) {
+        send.zipcode = tags_set['zip code'][0]
+      }
+
+      // if (tags_set.city.length !== 0) {
+      //   send.city = tags_set.city[0]
+      // }
+
+      // if (tags_set.type.length !== 0) {
+      //   send.typology = tags_set.type[0]
+      // }
+
+      // if (tags_set.style.length !== 0) {
+      //   send.style = tags_set.style[0]
+      // }
+
+      // if (tags_set.architect.length !== 0) {
+      //   send.architects = tags_set.architect[0]
+      // }
+
+      // if (tags_set.street.length !== 0) {
+      //   send.streets = tags_set.street[0]
+      // }
+
+      const data = await Api.searchData(send)
+      console.log(data)
+
+      window.localStorage.removeItem('building_data')
+      window.localStorage.removeItem('search_data')
+      window.localStorage.setItem('search_data', JSON.stringify(send))
+      window.localStorage.setItem('building_data', JSON.stringify(data))
+      if (window.location.hash !== '#/list') {
+        window.location.href = '/#/list'
+      }
+    })
+  },
+
+  addActive: (x, currentFocus) => {
+    /* a function to classify an item as "active": */
+    if (!x) return false
+    /* start by removing the "active" class on all items: */
+    SearchBar.removeActive(x)
+    if (currentFocus >= x.length) currentFocus = 0
+    if (currentFocus < 0) currentFocus = (x.length - 1)
+    /* add class "autocomplete-active": */
+    x[currentFocus].classList.add('autocomplete-active')
+  },
+
+  removeActive: (x) => {
+    /* a function to remove the "active" class from all autocomplete items: */
+    for (var i = 0; i < x.length; i++) {
+      x[i].classList.remove('autocomplete-active')
+    }
+  },
+
+  closeAllLists: (elmnt, inp) => {
+    /* close all autocomplete lists in the document,
+        except the one passed as an argument: */
+    var x = document.getElementsByClassName('autocomplete-items')
+    for (var i = 0; i < x.length; i++) {
+      if (elmnt !== x[i] && elmnt !== inp) {
+        x[i].parentNode.removeChild(x[i])
+      }
+    }
+  },
+
+  addItemsToObj: (arr, objArr) => {
+    if (arr.length !== 0) {
+      for (let i = 0; i < arr.length; i++) {
+        objArr.push(arr[i].name)
+      }
+    }
+  },
+
+  addItemsToList: (a, arr, name, icon = '', Nameclass = '', tagClass = '') => {
```

Comment:
Please keep some consistency in parameter name!
If you go for camelCase, `NameClass` should be `nameClass`.